sql/moz-fx-data-shared-prod/firefox_accounts_derived/nonprod_fxa_server_events_v1/query.sql

```diff
@@ -0,0 +1,22 @@
+SELECT
+  * REPLACE (
+    (
+      SELECT AS STRUCT
+        jsonPayload.* REPLACE (
+          (
+            SELECT AS STRUCT
+              jsonPayload.fields.* EXCEPT (device_id, user_id),
+              TO_HEX(SHA256(jsonPayload.fields.user_id)) AS user_id,
+              TO_HEX(SHA256(jsonPayload.fields.device_id)) AS device_id
+          ) AS fields
+        )
+    ) AS jsonPayload
+  ),
```

Comment:
Whilst `jsonPayload` is critical here, I'm not really sure about the other fields such as `labels` and `resource` since we do not use them anywhere in our analytics and I don't see how they could provide much value for this use case. I guess the main question to consider here would be if we actually should be preserving them at all. What are your thoughts @akkomar ?

Reply:
This is a good suggestion, but since there is a slight time pressure to get this work done I suggest we follow the convention used in production tables here and file a new PR to rework table schemas.

Reply:
At this stage I'd also argue that unless we implement this change for all the fxa log tables then maybe we should keep this one behaving the same way for consistency even if it's not ideal.

Reply:
That's fine with me, it was just a suggestion.

Reply:
Thanks, we do appreciate it. 🙇